### ISSUE TITLE
rs333

### ISSUE THREAD

**Issue by jnovembre** [NovembreLab/ggv#14]
This is Delta32 and doesn't seem to be showing up correctly (compare to the 1000 genomes data on rs333 online).  This may signal broader issues with our input data?

**Comment by danielecook** [comment]
The GGV API taps into an ensemble API to resolve rs positions using the following curl command:

```
curl 'http://grch37.rest.ens7mbl.org/variation/human/rs333' -H 'Content-type:application/json'
```

Which returns the following:
```
{
  "source": "Variants (including SNPs and indels) imported from dbSNP",
  "mappings": [
    {
      "location": "3:46414947-46414978",
      ...
      "allele_string": "GTCAGTATCAATTCTGGAAGAATTTCCAGACA/-",
      }
...
}
```

When specifying `rs333`, the API resolves to to chr3:46414961 which is actually [rs201797884](http://grch37.ensembl.org/Homo_sapiens/Variation/Population?db=core;r=3:46414461-46415461;v=rs201797884;vdb=variation;vf=49862778) which has 0.005 AF for CHB. Note that the pie chart does not to reflect the AF correctly. Try typing `rs201797884` into the API and it correctly resolves to this location.

<img width="248" alt="screen shot 2017-03-26 at 11 06 24 am" src="https://cloud.githubusercontent.com/assets/1536935/24333349/c0de7174-121b-11e7-9e11-48177a4d298b.png">

Based on the sequence and location of other indels, I was able to deduce that the reference for the 1K genomes file is `hg19`. __Is this correct?__

Closer examination of the 1K genomes dataset doesn't appear to have the CCR5 deletion variant:

```
./tabix static/data/1000genomes_phase3_table/ALL.chr3.phase3_shapeit2_mvncall_integrated.20130502.genotype.freq.gz 3:46404947-46424978 | awk 'length($5) != 1 || length($6) != 1 { print}'
```
* No large (31 bp) deletions are output.

## In Summary

- rs201797884 is a snp that appears in the within the deletion interval of the rs333 variant.
- queries for rs333 resolve to the correct position, but the rs333 deletion is __not__ present within the 1K genomes dataset.
- The API then uses the returned data for rs201797884.
- The pie graph is incorrectly displayed also.

On a side note, I was not able to edit the API and observe changes on the server. I'm not sure how it is implemented, but could use a little help there!

It would be good to see how these files were generated. I can investigate and see if the variant is present in the original dataset (I assume it is). __Any insight as to how 1KG files  were generated?__

**Comment by danielecook** [comment]
Ah I see the scale now for the rare variants - missed that before. Perhaps this can be made clearer. For example, a legend with the 5 colors used for the rare variants.
